roles/telemetry_logging/tasks/main.yml

```diff
@@ -0,0 +1,500 @@
+- name: Prepare Logging Tests
+  hosts: localhost
+  connection: local
+  gather_facts: false
+# vars:
+#   ansible_user: "{{ lookup('env', 'USER') }}"      
+#   APATH: "{{ lookup('env', 'PATH') }}"
+#   AHOME: "{{ lookup('env', 'HOME') }}"
+
+  #environment:
+  #  PATH: "{{ APATH }}:{{ AHOME }}/.local/bin/:/usr/local/bin/"
+
+  tasks:
+    - name: "Log into OCP"
+      ansible.builtin.shell:
+        cmd: |
+          oc login -u kubeadmin -p "12345678" https://api.crc.testing:6443
+      failed_when: false
+      changed_when: false
+
+
+- name: 'Verify projects exist'
+  hosts: localhost
+  connection: local
+  gather_facts: no
+  vars:
+     proj_out_file: verify_projects_exist_lresults.log
+     proj_list:
+       - openshift-openstack-infra
+       - openshift
+       - openstack-operators
+       - openshift-logging
+
+  tasks:
+    - name: Remove "{{ proj_out_file }}"
+      ansible.builtin.file:
+        path: "{{ proj_out_file }}"
+        state: absent
+      changed_when: false  
+
+    - name: Verify projects created
+      ansible.builtin.include_tasks: logging_proj_tests.yml
+      loop: "{{ proj_list }}"
+
+
+- name: Verify Openstack endpoints
+  hosts: localhost
```

Comment:
I am mistaken here. We may use ``localhost`` for these because of the unconventional way we execute the tests. i.e. we use a pattern of:

`- command: ansible-playbook ...`

However, the information about the zuul executer, the hostnames, etc are still accurate